PriceCalculations.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PriceCalculatorSolution
+{
+    public class PriceCalculations
+    {
+        public static int TaxPercentage;
+        public static int DiscountPercentage;
+        public PriceCalculations(int _taxPercentage, int _discountPercentage)
+        {
+            TaxPercentage = _taxPercentage;
+            DiscountPercentage = _discountPercentage;
+        }
+        public static decimal ProductTax(decimal _basicPrice)
+        {
```

Comment:
the method name should be a verb, as it does something, not represent an entity, I suggest CalculatePrice